apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -219,6 +230,61 @@ export class FilesStorageUC {
 		return fileRecord;
 	}
 
+	async copyFilesOfParent(
+		userId: string,
+		params: FileRecordParams,
+		copyFilesParams: CopyFilesOfParentParams
+	): Promise<Counted<FileRecord[]>> {
+		const [fileRecords, count] = await this.fileRecordRepo.findBySchoolIdAndParentId(params.schoolId, params.parentId);
+
+		if (count === 0) {
```

Comment:
I think a exception is to much on this place. If you think about copy course service, it will only call for each entity that they have a call to copy files if they exist. It is valid that nothing is found for a entity and we see in (empty) result.